.pre-commit-config.yaml

```diff
@@ -189,6 +189,7 @@ repos:
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
     - pytest
+    - requests
```

Comment:
Please add comments next to each line, explaining what uses the deps, just like in pylint down below.

Reply:
Thank you so much for the heads up about this one! Please let me know if what I've done looks correct. Pytest is a bit different than the others, so just let me know if anything needs to be revised.